stages/01_process.R

```diff
@@ -0,0 +1,42 @@
+library(purrr)
+library(arrow)
+library(fs)
+library(stringr)
+library(dplyr)
+library(tools)
+library(rvest)
+
+download_dir <- "download"
+data_dir <- "brick"
+fs::dir_create(data_dir)
+
+clean_html <- function(html) {
+    raw_text <- rvest::html_text(html)
+    raw_text
+}
+
+save_parquet <- function(file) {
+    print(file)
+    path <- fs::path_ext_remove(file) |>
+        fs::path_ext_set("parquet") |>
+        fs::path_file()
+    file_ext <- file_ext(file)
+    if (file_ext == "fa") {
+		print(file)
+        file.copy(from = file, to = paste0(data_dir, "/", basename(file)))
+    } else if (file_ext == "gff3") {
```

Comment:
Instead of removing the extension `.gff3`, can we make the final files something like `hsa.gff3.parquet` so that it is clear what is inside of the Parquet?